Tests/Functional/TestBase.php

```diff
@@ -154,4 +163,61 @@ protected function compareRecords($expectedRecords, $actualRecords, $checkAllFie
 
         $this->assertEquals($expectedRecords, $actualRecords);
     }
+
+    /**
+     * Executes an SQL file.
+     *
+     * @param Connection $conn
+     * @param string     $file
+     */
+    protected function executeSqlFile(Connection $conn, $file)
+    {
+        $sql = file_get_contents($file);
+        $stmt = $conn->prepare($sql);
+        $stmt->execute();
+    }
+
+    /**
+     * Executes large SQL file.
+     *
+     * @param string $filename
+     */
+    protected function executeLargeSqlFile($filename)
```

Comment:
Actually, having given this a second look, I started questioning the need for executeLargeSqlFile altogether. You are not saving any by memory using `file()` (in fact, you are losing a tiny bit), you are constructing sql statements line-by-line, which is something better left to Doctrine driver, and is ultimately unnecessary, as in the rare case where MySQL would complain about the packet size, either the test using the large sql statement can set the `max_allowed_packet` global MySQL variable, or the MySQL can be configured to accept larger packets. In the case where a file large enough to cause memory problems would be used, your solution would not help and we still would have to split that file into more acceptable chunks.

I would therefore suggest taking these lines:
````
$container = static::createClient()->getContainer();
/** @var EntityManager $manager */
$manager = $container->get('doctrine')->getManager();
$connection = $manager->getConnection(); 
```
and adding them to the executeSqlFile (in case $conn === null).